minetest/minetest
# DevTest: Fix broken PNG textures

If you started DevTest lately, you got these annoying console/log warning multiple times:

```
libpng warning: bKGD: invalid index

```

Turns out a couple of textures in the `basetools` mod were broken, so I fixed them.

## How to test

1. Launch DevTest
2. See if any texture warnings show up in log
